### PR TITLE
Build pet profile

### PR DESCRIPTION
**Pull request by acheung-94**

### CODE REVIEW

File: frontend/src/components/PetProfile/PetProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+const PetProfile = () => {
+    const { petId } = useParams()
+    const pet = useSelector(selectPet(parseInt(petId)))
+    const petAttributes = Object.entries(pet)
+
+    const renderAttributes = () => {
+        return petAttributes.map( (pair, idx) => {
+            let key = pair[0]
+            let val = pair[1]
```
Comment (ChristopherJTrent) [inline frontend/src/components/PetProfile/PetProfile.jsx]: `return petAttributes.map( ([key, val], idx) => {`
instead, perhaps?

**Review by ChristopherJTrent — CHANGES_REQUESTED** [review]
only one minor quibble, other than that this looks good

File: frontend/src/components/PetProfile/PetProfile.jsx (hunk 1 above)
Comment (acheung-94) [reply frontend/src/components/PetProfile/PetProfile.jsx]: ah! that's what i was trying to do. ty!

Comment (ChristopherJTrent) [inline frontend/src/components/PetProfile/PetProfile.jsx]:
```suggestion
        return petAttributes.map( ([key, val], idx) => {
```

**Review by ChristopherJTrent — APPROVED** [review]
LGTM

**Pull request merged**